src/Eshopworld.Messaging/ServiceBusExtensions.cs

```diff
@@ -34,7 +34,7 @@ await sbNamespace.Queues
                              .CreateAsync();
 
             await sbNamespace.RefreshAsync();

```

Comment:
so this should not be necessary anymore then if you are not enumeration to locate the topic/queue etc..

Reply:
Hey Oisin,
Just to confirm - I can go ahead and remove the `await sbNamespace.RefreshAsync();`? as this has been made redundant?

Thanks

Reply:
will call you in a bit

Reply:
I can confirm that the refresh is not necessary.  The namespace is up to date following the create call.  Oisin noticed this in the SDK samples, and my tests confirm it (there's a get entity before create, then create entity, then get entity after, all of which work as expected).